docs/core/deploying/ready-to-run.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Ready to Run
+description: Learn what ReadyToRun is and why you should consider using it as part of the deployment.
+author: davidwr
+ms.author: davidwr
+ms.date: 09/21/2020
+---
+# ReadyToRun Compilation
+Startup time and application latency can be improved by compiling your application assemblies as ReadyToRun (R2R) format. R2R is a form of ahead-of-time (AOT) compilation.
+
+R2R binaries improve startup performance by reducing the amount of work the just-in-time (JIT) compiler needs to do as your application loads. The binaries contain similar native code compared to what the JIT would produce. However, R2R binaries are larger because they contain both intermediate language (IL) code, which is still needed for some scenarios, and the native version of the same code. R2R is only available when you publish an app that targets specific runtime environments (RID) such as Linux x64 or Windows x64.
+
+To compile your project as ReadyToRun, do the following:
+
+01. Add the `<PublishReadyToRun>` setting to your project:
```

Comment:
Does this also need to mention that optionally /p:PublishReadyToRun could be passed to `dotnet publish` too?